app/src/androidTest/java/com/sdp/movemeet/backend/providers/AuthenticationInstanceProviderTest.java

```diff
@@ -15,7 +19,7 @@
 import static org.mockito.Mockito.mock;
 
 @RunWith(AndroidJUnit4.class)
-public class AuthentificationInstanceProviderTest {
+public class AuthenticationInstanceProviderTest {
```

Comment:
Nice, orthographically more correct like this! 👍

Reply:
Nice catch!